content/docs/getting-started/app-maintenance.md

```diff
@@ -15,6 +15,10 @@ Run `cf restage <app-name>` periodically (for example, every two weeks) to get a
 
 ## Keeping your app up to date
 
-You’re responsible for keeping your own app up to date. `cf restage` restarts your app with updates from the platform but not any updates to the software you use. If you’re running Drupal, Rails, Django, etc., you’re responsible for patching and updating those components and anything built on top of them. Only you should control when and how your apps are maintained. We encourage you to keep your apps, any associated libraries, and custom buildpacks up to date and make a plan for testing those changes in a [staging or development environment](https://cloud.gov/docs/getting-started/concepts/#spaces).
+You’re responsible for keeping your own app up to date. `cf restage` rebuilds and restarts your app with updates from the platform, but not any updates to the software you use. If you’re running Drupal, Rails, Django, etc., you’re responsible for patching and updating those components and anything built on top of them. Only you should control when and how your apps are maintained. We encourage you to keep your apps, any associated libraries, and custom buildpacks up to date and make a plan for testing those changes in a [staging or development environment](https://cloud.gov/docs/getting-started/concepts/#spaces).
+
+Whenever you deploy an app update with `cf push`, your app will use the latest buildpack (if you’re using one provided by the platform) and filesystem.
+
+## Avoiding downtime during buildpack or app updates
```

Comment:
Do we want to make a stronger statement here? "We recommend you setup zero downtime deploys" vs. "your app may experience downtime, so you should probably do this"?

Reply:
Based on customer experience so far, this level of recommendation seems fine to me - ensuring people know it's an option if they need it. People are likely to get a sense during their development and testing processes of whether they need to set up the additional complexity of a zero-downtime deployment plugin.